airqo-platform/AirQo-api
# Multi hop

WHAT DOES THIS PR DO?
Creates an endpoint for computing calibrated values using the multi-hop approach

WHAT JIRA STORY/TASK IS RELATED TO THIS PR?
https://airqoteam.atlassian.net/jira/software/projects/PLAT/boards/41?assignee=60095b78dfb0c70069357e54&selectedIssue=PLAT-523 

WHAT IS THE LINK TO THE PR BRANCH
https://github.com/airqo-platform/AirQo-api/tree/Multi-hop

HOW DO I TEST OUT THIS PR?
cd AirQo-api/src/calibrate-multi-hop
python -m venv [DESTINATION_FOLDER]
source [DESTINATION_FOLDER]/Scripts/activate
pip install -r requirements.txt
flask run

WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:
[POST] http://localhost:4001/api/v1/calibrate/multihop

Sample Input body -
{
"datetime":"2020-09-06 12:56:59+00:00",
"raw_values": [
{
"raw_value": 100,
"sensor_id": 832251
},
{
"raw_value": 22,
"sensor_id": 82720
},
{
"raw_value": 29,
"sensor_id": 870144
}
]
}

IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?
N/A

ARE THERE ANY RELATED PRs?
N/A
